chat.go

```diff
@@ -206,6 +208,14 @@ func (api *Client) SendMessageContext(ctx context.Context, channelID string, opt
 		return "", "", "", err
 	}
 
+	reqBody, err := ioutil.ReadAll(req.Body)
```

Comment:
I think it is better for debugging but it may affect performance by `iotuil.ReadAll`.
How about you use `if api.Debug() { ... }` ?

Reply:
It makes perfect sense. Change is on the way! 👍

Reply:
For the record, I realise the `api.Debugf()` checks if the debug is enabled but I think it's more readable if it's inside of the if. :)